src/planscape/plan/views.py

```diff
@@ -34,23 +34,14 @@ def create_plan(request: HttpRequest) -> HttpResponse:
         if region_name is None:
             raise ValueError("Unknown region_name: " + region_name_input)
 
-        # Get the geometry of the plan.  Convert it to a MultiPolygon
-        # if it is a simple Polygon, since the model column type is
-        # MultiPolygon.
+        # Get the geometry of the plan.
         geometry = body.get('geometry', None)
         if geometry is None:
             raise ValueError("Must specify geometry")
-        features = geometry.get('features', [])
-        if len(features) > 1 or len(features) == 0:
-            raise ValueError("Must send exactly one feature.")
-        feature = features[0]
-        geom = feature['geometry']
-        if geom['type'] == 'Polygon':
-            geom['type'] = 'MultiPolygon'
-            geom['coordinates'] = [feature['geometry']['coordinates']]
-        geometry = GEOSGeometry(json.dumps(geom))
-        if geometry.geom_type != 'MultiPolygon':
-            raise ValueError("Could not parse geometry")
+        # Convert to a MultiPolygon if it is a simple Polygon, since the model column type is
+        # MultiPolygon.
+        # If this fails, the rest of the clause is skipped.
+        geometry = _convert_polygon_to_multipolygon(geometry)
```

Comment:
Would it be useful to check that the geometry is valid?
I've created invalid geometries before (though I assume the FE drawing app will only emit valid geometries).

https://docs.djangoproject.com/en/4.1/ref/contrib/gis/geos/#django.contrib.gis.geos.GEOSGeometry.valid

Reply:
and just for my understanding: how is the geometry's srid set?
I've created geometries without assigning srid's before, which causes the forsys backend to raise errors.

Reply:
1) _convert_polygon_to_multipolygon does some validation. See the test file for associated checks. Does that cover everything you were thinking of or are there other checks you had in mind?

2) It doesn't seem like we need to set an SRID to save it to the DB (see test file). Is the SRID used for something special in forsys?

Reply:
I see!

Yes: srid will be necessary to check whether a raster element falls within a project area geometry.

Reply:
Let's keep discussing in your PR. I also left a question about it there